src/components/PersonItems/PersonItem.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+interface Props {
+  person: Person;
+  people: Person[];
+}
+export const PersonItem: React.FC<Props> = ({ person, people }) => {
+  const { personId } = useParams();
+  const { motherName, fatherName } = person;
+
+  const motherNameFounded = people.find(human => human.name === motherName);
+  const fatherNameFounded = people.find(human => human.name === fatherName);
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': personId === person.slug,
+      })}
+      key={person.slug}
```

Comment:
Keys only make sense in the context of the surrounding array. 
You've already mentioned it before
''PersonItem
     key={person.slug}''